projects/plugins/boost/app/assets/src/js/utils/analytics.ts

```diff
@@ -1,18 +1,38 @@
-export function recordBoostEvent(
+type RecordSuccess = {
+	success: boolean;
+};
+
+export async function recordBoostEvent(
 	eventName: string,
 	eventType: string,
```

Comment:
We're not using the `eventType` argument here. Doesn't have to be a part of this PR, but it would be nice to eliminate this unused argument across the project. Should be a 2 minute search & modify.